.ci/generate_bear_requirements.py

```diff
@@ -48,6 +48,7 @@
     'GemRequirement',
     'ComposerRequirement',
     'CabalRequirement',
+    'RRequirement',
```

Comment:
The bears of R language have `RscriptRequirement` dependency which should be here, not `RRequirement`